client/me/concierge/shared/confirmation.js

```diff
@@ -11,35 +11,33 @@ import React, { Component } from 'react';
 import Button from 'components/button';
 import Card from 'components/card';
 import FormattedHeader from 'components/formatted-header';
-import { localize } from 'i18n-calypso';
 
-class ConfirmationStep extends Component {
+class Confirmation extends Component {
 	render() {
-		const { site, translate } = this.props;
+		const { site, confirmationButton, confirmationDescription, confirmationTitle } = this.props;
+
 		return (
 			<Card>
 				<img
-					className="concierge__confirmation-illustration"
+					className="shared__confirmation-illustration"
```

Comment:
Any reason not to leave this unchanged?

Reply:
The parent folder prefix is required for a successful linting validation.